common/src/test/scala/org/neo4j/spark/util/Neo4jImplicitsTest.scala

```diff
@@ -105,4 +107,19 @@ class Neo4jImplicitsTest {
 
     assertEquals(Set[String]("hero_name"), struct.getMissingFields(Set("is_hero", "hero_name")))
   }
+
+  @Test
+  def `getMissingFields should handle maps`: Unit = {
+    val struct = StructType(Seq(
+      StructField("im", DataTypes.StringType),
+      StructField("im.a", DataTypes.createMapType(DataTypes.StringType, DataTypes.StringType)),
+      StructField("im.also.a", DataTypes.createMapType(DataTypes.StringType, DataTypes.StringType)),
+      StructField("im.not.a.map", DataTypes.StringType),
+      StructField("fi``(╯°□°)╯︵ ┻━┻eld", DataTypes.StringType)
+    ))
+
+    val result = struct.getMissingFields(Set("im.aMap", "`im.also.a`.field", "`im.a`.map", "`im.not.a.map`", "fi``(╯°□°)╯︵ ┻━┻eld"))
+
+    assertEquals(1, result.size)
```

Comment:
should we check which one is the missing field? just to avoid false positives in the future

Reply:
```scala
assertEquals("im.aMap", result(0))
```

I guess something like this

Reply:
Yes it's better let me do it

Reply:
Done